hooks/command

```diff
@@ -42,8 +42,10 @@ while IFS=$'\n' read -r option ; do
   options+=("$option")
 done < <(plugin_read_list OPTIONS)
 
+VERSION="${VERSION:-latest}"
```

Comment:
Also, it might be better to make this an option and use `BUILDKITE_PLUGIN_SHELLCHECK_VERSION`

Reply:
> `stable` might be a better default

Agreed. Will update.

> Also, it might be better to make this an option and use `BUILDKITE_PLUGIN_SHELLCHECK_VERSION`

Ah, I think that is where I was missing some information on what/how to update. Couldn't see what did/didn't apply the prefix.